apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -648,12 +665,14 @@ export function WriteInsAdjudicationScreen(): JSX.Element {
                       key={candidate.id}
                       ref={i === 0 ? firstAdjudicationButton : undefined}
                       variant={isCurrentAdjudication ? 'secondary' : 'regular'}
-                      onPress={async () => {
-                        if (!isCurrentAdjudication) {
-                          await adjudicateAsOfficialCandidate(candidate);
+                      onPress={() => {
+                        if (
+                          isWriteInAdjudicationContextFresh &&
+                          !isCurrentAdjudication
+                        ) {
+                          adjudicateAsOfficialCandidate(candidate);
                         }
                       }}
-                      disabled={!isWriteInDetailViewFresh}
```

Comment:
So, we want to prevent adjudication while the `AdjudicationContext` is not fresh. Previously we would disable the button, but I think it looked a little jarring to have the buttons flash disabled (dotted outline) and back. So I've just made the buttons no-op for that split second. I think it would be hard to hit the button at that time.